### DISCUSSION TITLE
Dietpi-Software | LinkAce - Bookmark archive

### DISCUSSION THREAD

**Discussion by zz5zz** [MichaIng/DietPi#6190] [Software Request]
#### Formal software information
- Software title | LinkAce
- Short description | A bookmark archive with automatic backups to the Internet Archive, link monitoring, and a full REST API
- Official URL (if available) | https://www.linkace.org/
- GitHub URL (if available) | https://github.com/Kovah/LinkAce/
- (Official) install documentation | https://www.linkace.org/docs/v1/setup/setup-without-docker/

#### Are there similar/alternative software titles available with DietPi-Software?
None with bookmark functionality

#### What makes your requested software better than the above solutions, if available?
This is one of the most built out Bookmark servers out there, similar software can be found here:
https://github.com/awesome-selfhosted/awesome-selfhosted#bookmarks-and-link-sharing

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
The installation is very high level and requires deeper insight into DietPi to be able to install it.
DietPi could automate the installation and the software can get a setup that integrates with everything else and gets it's own dedicated URL and port. This will save a lot of time and make it easier for non experienced Linux users.

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. Install the webserver stack (Apache or Nginx)
2. Install all the required PHP modules
3. Install MySQL
4. Edit the .env file
5. Generate a secret key
6. Update Apache configuration

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? 
Yes, I could help out with this

**Comment by Joulinar** [comment]
Many thanks for your request.

I converted it into a discussion, so people can vote for it via ⬆️ button at above post.
